### DISCUSSION TITLE
Sample project creation permission issue

### DISCUSSION THREAD

**Discussion by karattejoseph** [Budibase/budibase#4778] [Help]
I am starting the sample CRUD project as per the docs. After I created the layout and run it, it is showing "You don't have permission to use this app". When I checked the documentation there is a Manage access dropdown, which is not present in my budibase instance. I have hosted the docker image in digital ocean. Any idea about it

**Comment by mjashanks** [comment]
> After I created the layout and run it

I'm not sure what you mean by this. Are you previewing the app that you have just built? Are you trying to run a published app?

What does the message  "You don't have permission to use this app" look like? Can you screenshot that?

**Reply by karattejoseph** [reply]
Yes, I am previewing the app, just built by publish it. 
![budibase-error](https://user-images.githubusercontent.com/13979896/156601252-470deba2-ac08-4c06-ade7-04ccd7faf21e.JPG)
![budibase-studio](https://user-images.githubusercontent.com/13979896/156601256-cfc09dab-6f57-4e88-91c7-651f31c6d6e7.JPG)

**Reply by mjashanks** [reply]
Thanks. It'd be really helpful if you could
1. Open your browser's dev tools, whilst in the preview
2. Goto the network tab
3. Reload the page

Look for any failed requests (in red) and report back with some screenshots, especially of the "Response" tab.

**Reply by karattejoseph** [reply]
![app_load_error](https://user-images.githubusercontent.com/13979896/156612962-5b5c54a0-0869-4f02-b921-6ed1b0e3cb21.JPG)
![network error](https://user-images.githubusercontent.com/13979896/156612966-27ada49f-02dd-402b-b11a-69943e3bcf0c.JPG)
These are the errors

**Reply by mjashanks** [reply]
Thanks - and do you have any errors in your `Console` tab?

**Reply by karattejoseph** [reply]
Only that favicon error

**Reply by mjashanks** [reply]
It's a strange one. You could try exporting your app, then importing it again - this will create a brand new app, and might re-build it.

**Comment by karattejoseph** [comment]
I tried by creating a new app and now tried with importing and exporting
existing app also. Same results, do I go for reinstallation or something ?

On Fri, Mar 4, 2022 at 3:10 PM Michael Shanks ***@***.***>
wrote:

> It's a strange one. You could try exporting your app, then importing it
> again - this will create a brand new app, and might re-build it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/4778#discussioncomment-2294056>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKVB6CATGZISATUC6CO64LU6HLBLANCNFSM5P22FBPQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Joseph Alexander
Ph:9562291003
Linked in:http://in.linkedin.com/in/josephisready
twitter:http://twitter.com/josephisready
